bin/run-ldc93s1.sh

```diff
@@ -1,16 +1,18 @@
 #!/bin/sh
 set -xe
-
 if [ ! -f DeepSpeech.py ]; then
     echo "Please make sure you run this from DeepSpeech's top level directory."
     exit 1
 fi;
 
+XDG_DATA_HOME=${XDG_DATA_HOME:="$HOME/DeepSpeech"}
+
 python -u DeepSpeech.py \
   --importer ldc93s1 \
   --train_batch_size 1 \
   --dev_batch_size 1 \
   --test_batch_size 1 \
   --n_hidden 494 \
   --epoch 50 \
+  --checkpoint_dir "$XDG_DATA_HOME/$(basename ${0%.*})" \
```

Comment:
This contains the `run-` prefix which I think is not desirable. Ideally you could do this in one step
```bash
...
--checkpoint_dir "$XDG_DATA_HOME/$(basename -s .sh ${0#run-})" \
...
```
But that doesn't seem to work due to some bash foo I don't have.

So one can do it in two steps.
```bash
XDG_DATA_HOME=...
CHECKPOINT_DIR=$(basename ${0%.*})
CHECKPOINT_DIR=${CHECKPOINT_DIR#run-}
...
   --validation_step 5 \
   --checkpoint_dir "$XDG_DATA_HOME/$CHECKPOINT_DIR" \
...
```